crates/ruff_formatter/src/printer/mod.rs

```diff
@@ -108,7 +108,7 @@ impl<'a> Printer<'a> {
                     self.flush_line_suffixes(queue, stack, Some(element));
                 } else {
                     // Only print a newline if the current line isn't already empty
-                    if self.state.line_width > 0 {
+                    if self.state.line_width != 0 {
```

Comment:
No. It's a u32 and must be a zero or a positive number. But I should probably revert this change. It's unrelated. I only changed it because I first changed the printer to differentiate between no-content and a line width zero-width content. But I now opted to not make this change, so I should not change the printer here (I'm pretty sure LLVM compiles this down to the same code anyway)